repository/SmalltalkCI-Pharo-Core.package/SmalltalkCIPharo9.class/instance/preLoad.st

```diff
@@ -0,0 +1,10 @@
+executing
+preLoad
+	[ (IceRepositoryCreator new
+		location: projectDirectory asFileReference;
+		createRepository) register. ] 
+		on: Error 
+		do: [ :e |
+		      	 self class printWarningToStdout: 'Could not register the tested repository with Iceberg.'.
```

Comment:
Can this just be a long string instead of multiple calls to `self class printWarningToStdout`?